### PR TITLE
Make.go improvements

### PR DESCRIPTION
**Pull request by tboerger**

### CODE REVIEW

File: make.go [hunk 1]
```diff
@@ -283,19 +356,44 @@ func clean() error {
...
+// run is a helper function that executes commands
+// and assigns stdout and stderr targets
+func run(command string, args ...string) error {
+	cmd := exec.Command(command, args...)
+
+	cmd.Stdout = os.Stdout
+	cmd.Stderr = os.Stderr
+
+	trace(cmd.Args)
+	err := cmd.Run()
```
Comment (bradrydzewski) [inline make.go]: we can tighten this up -- `return cmd.Run()`  :)

File: make.go [hunk 2]
```diff
@@ -55,15 +53,46 @@ func main() {
 
 type step func() error
 
+func executeDeps() error {
+	bindata := run(
+		"go",
```
Comment (bradrydzewski) [inline make.go]: we could probably use a slice, similar to the css files:

```
deps := []string{"github.com/jteeuwen/go-bindata/...", "golang.org/x/tools/cmd/cover" }
for _, dep := range deps {
   err := run("go", "get", "-u", dep)
   if err != nil {
    return err
  }
}
```

File: make.go [hunk 3]
```diff
@@ -145,104 +174,148 @@ func styles() error {
...
 	}{
-		{"./bin/drone-agent", "drone/drone-agent"},
-		{"./bin/drone-server", "drone/drone"},
+		{
+			"bin/drone-agent",
+			"drone/drone-agent"},
+		{
+			"bin/drone-server",
+			"drone/drone"},
 	}
```
Comment (bradrydzewski) [inline make.go]: can you run `gofmt` on the code? the wrapping seems a bit inconsistent is all:

```
+		{
+			"bin/drone-server",
-			"drone/drone"},
+			"drone/drone",
+		}
``

File: make.go (hunk 1 above)
Comment (tboerger) [inline make.go]: Haha, yes... you are right. I will fix that.

File: make.go (hunk 2 above)
Comment (tboerger) [inline make.go]: Good idea, I will do that.

**Comment by bradrydzewski** [comment]
just some minor comments above, but otherwise great work. Would you also mind updating the build instructions in the README? https://github.com/drone/drone/tree/0.4.0#cloning-building-running

**Comment by tboerger** [comment]
> just some minor comments above, but otherwise great work. Would you also mind updating the build instructions in the README? https://github.com/drone/drone/tree/0.4.0#cloning-building-running

Sure, I will address that in the next commit.

**Comment by tboerger** [comment]
@bradrydzewski I have addressed all the comments. Should be fine now :)

**Pull request merged**